### PR TITLE
Update the stage for RecursiveReadOnlyMounts gate in v1.31

### PR DESCRIPTION
**Pull request by tengqm**
The RecursiveReadOnlyMounts feature has graduated into Beta.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/approve

File: content/en/docs/reference/command-line-tools-reference/feature-gates/RecursiveReadOnlyMounts.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/RecursiveReadOnlyMounts.md:1]: I disagree with the filename change.

/approve cancel

**Comment by AkihiroSuda** [comment]
ping @tengqm

**Pull request closed without merging**